sycl/tools/sycl-vxx/bin/sycl_vxx.py

```diff
@@ -236,6 +237,8 @@ def _get_compile_kernel_cmd_out(self, kernel):
             "--save-temps", "-c", "-k", kernel['name'], '-o', kernel_output,
             self.vpp_llvm_input
         ]
+        if comp_config is not None and path.isfile(comp_config):
+            command.extend(("--config", comp_config))
```

Comment:
Converting path to absolute can be better for debugging purposes (the debug can occur from any directory).

Reply:
I doubt that anyone would provide a relative path here, but anyway, will something like this be okay for you?
```suggestion
            command.extend(("--config", str(Path(comp_config).resolve())))
```

Reply:
The `str()` is not required (the conversion is performed by `subrocess.run()`).

I think a relative path can occur if the compilation line is given along a self-contained program source and parameter file (for instance for reproducibility tests).